### ISSUE TITLE
redub test fails with error: hipjson.d(213): Tried to iterate a non array object of type string

### ISSUE THREAD

**Issue by nordlow** [MrcSnm/redub#32]
In local copy of https://github.com/nordlow/yyjson-d at 4d9c847caa7067ecc4245ef6242c824e4d7ee309

`redub -v test` fails as

```
Parsing /home/per/Work/yyjson-d/dub.sdl at /home/per/Work/yyjson-d with : -c 
Added project yyjson-d to memory cache.
object.Exception@../../.dub/packages/redub/1.16.1/redub/hipjson/source/hipjson.d(213): Tried to iterate a non array object of type string
----------------
??:? [0x59ef7211672e]
??:? [0x59ef72116392]
??:? [0x59ef7213d8be]
??:? [0x59ef7211e9ff]
??:? [0x59ef72039287]
??:? [0x59ef720bfa2a]
??:? [0x59ef72035d32]
??:? [0x59ef72038562]
??:? [0x59ef720c0b75]
??:? [0x59ef7211d2dd]
??:? [0x59ef720c0b04]
??:? [0x59ef720330dc]
??:? [0x59ef7202aa79]
??:? [0x59ef7202fd98]
??:? [0x59ef72087e13]
??:? [0x59ef720870ee]
??:? [0x59ef72086bf8]
??:? [0x59ef7211e6dc]
??:? [0x59ef7211e5d6]
??:? [0x59ef7211e42c]
??:? [0x74d23ba2a1c9]
??:? __libc_start_main [0x74d23ba2a28a]
??:? [0x59ef72028c04]
Error Program exited with code 1
```

**Comment by MrcSnm** [comment]
I'll fix that with the major new release of redub

**Comment by nordlow** [comment]
Thank you.

**Comment by MrcSnm** [comment]
Fixed in v1.17.0
